script/Arcueid/Day 11/11_00_ARC11_3B.txt

```diff
@@ -351,6 +351,7 @@ Her golden hair dances softly before me.
 [sha:d0a27bec96943dd5b5f7aa553dc92cbc647dfef0]{
 -- Page 275, Offset 30313.
 -- 「―――ううん。わたし、すごくどきどきしてる」
+// group pr check: flustered not accurate
```

Comment:
Personally think it's kinda fine, I associate being flustered with being flushed, anxious, high heart rate

Reply:
as do I.